testing/btest/scripts/base/utils/dir.test

```diff
@@ -1,4 +1,9 @@
 # @TEST-EXEC: btest-bg-run zeek zeek -b ../dirtest.zeek
+#
+# The following helps eliminate race conditions when Zeek startup is
+# slow due to ZAM compilation.
+# @TEST-EXEC: sleep 5
```

Comment:
I'll remove this unconditional `sleep 5` and increase below 10 seconds to 20 seconds. Effect should be similar.